tests/setupTests.ts

```diff
@@ -1,5 +1,3 @@
-import 'jest-localstorage-mock';
```

Comment:
With the mock in place it was no possible to make the test fail without the fix.